modules/ROOT/pages/updates-upgrades-rollbacks.adoc

```diff
@@ -50,6 +51,10 @@ $ rpm-ostree rebase fedora:fedora/{version}/x86_64/{variant}
 
 The process is very similar to a system update: the new OS is downloaded and installed in the background, and you just boot into it when it is ready.
 
+Additionally, you can choose to rebase to a different variant of Silverblue, like for example Fedora Kinoite. Kinoite is similar to Silverblue, except for the fact that it uses the KDE Plasma desktop environment instead of the default Gnome desktop environment. 
```

Comment:
I would replace "different variant of Silverblue" with "different immutable variant of Fedora". Also replace Gnome with GNOME.